src/design-pages/passkey_kerberos.rst

```diff
@@ -0,0 +1,223 @@
+Passkey authentication Kerberos integration
+=======================================================
+
+Related ticket(s):
+------------------
+ * https://github.com/SSSD/sssd/issues/6228
+
+Problem statement
+-----------------
+This design is an extension of the "enable passkey authentication", where a
+centrally managed user can authenticate locally in a system with a passkey
+token. This might not be enough for SSSD in some environments like big
```

Comment:
Please change this sentence to simply state that this design describes Kerberos pre-authentication mechanism using passkey authentication. Once Kerberos pre-authentication complete, a received initial ticket granting ticket (TGT) can be used for signle sign-on into other Kerberos-enabled services.

Reply:
Changed